src/WebJobs.Extensions.DurableTask/HttpApiHandler.cs

```diff
@@ -546,16 +546,36 @@ protected virtual DurableOrchestrationClientBase GetClient(OrchestrationClientAt
             return this.config.GetClient(attribute);
         }
 
+        internal HttpCreationPayload GetInstanceCreationLinks()
+        {
+            this.ThrowIfWebhooksNotConfigured();
+
+            Uri notificationUri = this.config.Options.NotificationUrl;
+
+            string hostUrl = notificationUri.GetLeftPart(UriPartial.Authority);
+            string baseUrl = hostUrl + notificationUri.AbsolutePath.TrimEnd('/');
+            string instancePrefix = baseUrl + OrchestratorsControllerSegment + "{functionName}[/{instanceId}]";

```

Comment:
Does Durable JS actually consume these URLs? If so, is the idea that it will parse out the square brackets?